__tests__/posts.ts

```diff
@@ -2011,6 +2011,126 @@ describe('mutation sharePost', () => {
+      role: SourceMemberRoles.Member,
+    });
+    await con.getRepository(SharePost).save({
+      id: 'sharePost',
+      shortId: 'sharePost',
+      sourceId: 's1',
+      type: PostType.Share,
+      title: 'Foo Bar',
+      authorId: '1',
+    });
+  });
+
+  it('should not authorize when not logged in', () =>
+    testMutationErrorCode(
+      client,
+      { mutation: MUTATION, variables },
+      'UNAUTHENTICATED',
+    ));
+
+  it('should throw error when post does not exist', async () => {
+    loggedUser = '1';
+    return testMutationErrorCode(
+      client,
+      { mutation: MUTATION, variables: { ...variables, id: 'nope' } },
+      'NOT_FOUND',
+    );
+  });
+
+  it('should restrict member when user is not the author of the post', async () => {
+    loggedUser = '2';
+
+    return testMutationErrorCode(
+      client,
+      { mutation: MUTATION, variables },
+      'FORBIDDEN',
+    );
+  });
+
+  it('should update the post w/ a trimmed commentary', async () => {
+    loggedUser = '1';
+    const res = await client.mutate(MUTATION, {
+      variables: { ...variables, commentary: '  My comment  ' },
+    });
+    expect(res.errors).toBeFalsy();
+    const post = await con
+      .getRepository(SharePost)
+      .findOneBy({ id: variables.id });
+    expect(post.title).toEqual('My comment');
+  });
+
+  it('should share to squad without commentary', async () => {
+    loggedUser = '1';
+    const res = await client.mutate(MUTATION, {
+      variables: { ...variables, commentary: null },
+    });
+    expect(res.errors).toBeFalsy();
+    const post = await con
+      .getRepository(SharePost)
+      .findOneBy({ id: variables.id });
+    expect(post.title).toBeNull();
+  });
+
+  it('should share to squad with mentioned users', async () => {
+    loggedUser = '1';
+    await con.getRepository(User).update({ id: '2' }, { username: 'lee' });
+    const params = { ...variables };
+    params.commentary = 'Test @lee @non-existent';
+    const res = await client.mutate(MUTATION, { variables: params });
+    expect(res.errors).toBeFalsy();
+    const post = await con
+      .getRepository(SharePost)
+      .findOneBy({ id: variables.id });
+    expect(post.authorId).toEqual('1');
+    expect(post.titleHtml).toMatchSnapshot();
+    const mentions = await con
+      .getRepository(PostMention)
+      .findOneBy({ mentionedUserId: '2', mentionedByUserId: '1' });
+    expect(mentions).toBeTruthy();
+  });
+
+  it('should escape html content on the title', async () => {
+    loggedUser = '1';
+    await con.getRepository(User).update({ id: '2' }, { username: 'lee' });
+    const params = { ...variables };
+    params.commentary = `<style>html { color: red !important; }</style>`;
+    const res = await client.mutate(MUTATION, { variables: params });
+    expect(res.errors).toBeFalsy();
+    const post = await con
+      .getRepository(SharePost)
+      .findOneBy({ id: variables.id });
+    expect(post.titleHtml).toMatchSnapshot();
```

Comment:
I know we probably do it for the other one, but can you perhaps change this to inline evaluation?
Slowly want to get rid of snapshots as they always turn to be pests.